project_info/__init__.py

```diff
@@ -1,12 +1,19 @@
-from django.conf import settings
-import json
 import os.path
+import json
+
+
+def get_project_info_dict(pip_config=None):
```

Comment:
musím umožnit nastavovat cestu k souboru, pokud to chci vytáhnout ze settings (problém slepice/vejce)